plugins/quay/README.md

```diff
@@ -1,8 +1,22 @@
 # Quay plugin for Backstage
 
-The Quay plugin displays the information about your container images within the Quay registry.
+The Quay plugin displays the information about your container images within the Quay registry in your Backstage application.
 
-## Using Quay plugin
+# Table of contents
+
+1. [For administrators](#for-administrators)
+
+   a. [Installation](#installation)
+
+   b. [Development](#development)
+
+1. [For users](#for-users)
+
+   a. [Using Quay plugin in Backstage](#using-quay-plugin-in-backstage)
```

Comment:
It's not clear to me whether this article was omitted on purpose or not:
```suggestion
   a. [Using the Quay plugin in Backstage](#using-quay-plugin-in-backstage)
```

Reply:
Added the article.